src/main/scala/net/snowflake/spark/snowflake/SnowflakeTelemetry.scala

```diff
@@ -435,7 +454,12 @@ object SnowflakeTelemetry {
     "spark.sql.ansi.enabled",
     "spark.pyspark.driver.python",
     "spark.pyspark.python",
-    "spark.sql.session.timeZone"
+    "spark.sql.session.timeZone",
+    // If the users need to use Pandas DataFrame with Spark DataFrame,
+    // spark.sql.execution.arrow.pyspark.enabled needs to be set to "true"
```

Comment:
Add these 3 options to white list.

If users needs to use Pandas DataFrame, for example, convert a Spark DataFrame to a Pandas DataFrame,
They need to enable "spark.sql.execution.arrow.pyspark.enabled" . 
So I think we can add them.